cabal-install/Distribution/Client/Upload.hs

```diff
@@ -159,10 +160,19 @@ handlePackage :: HttpTransport -> Verbosity -> URI -> Auth
 handlePackage transport verbosity uri auth path =
   do resp <- postHttpFile transport verbosity uri path auth
      case resp of
-       (200,_)     ->
-          notice verbosity "Ok"
+       (code,warnings) | code `elem` [200, 204] -> do
```

Comment:
@23Skidoo Hmm, it means `204 NoContent` and seems like it is only sent when uploading documentation for a package candidate, so it is not necessary in this particular case since the function is only used for packages. But since there is similar code in the function to upload docs, i thought thtat the response code makes sense here as well. Perhaps we should just accept any 20x code, since they all designate a successful operation?